src/vaadin-combo-box-mixin.html

```diff
@@ -623,10 +624,21 @@
       }
     }
 
+    _itemLabelPathChanged(itemLabelPath, oldItemLabelPath) {
+      if (typeof itemLabelPath !== 'string') {
+        console.error('You should set itemLabelPath to a valid string');
+      }
+
+      if (this.selectedItem) {
+        this._inputElementValue = this._getItemLabel(this.selectedItem);
+      }
+    }
+
     _filterChanged(filter, itemValuePath, itemLabelPath) {
-      if (filter === undefined || itemValuePath === undefined || itemLabelPath === undefined) {
```

Comment:
Why was this line changed? Especially, `itemValuelPath` change seems unrelated.
Same as above: please revert the change to this line in case if new logic does not require it.

Reply:
Discussed in person, OK to include this change.